isites_migration/scripts/enrollment_viewer_manager.sql

```diff
@@ -0,0 +1,3 @@
+GRANT SELECT ON enrollment_viewer_manager TO termtool;
```

Comment:
Where would we normally (I use that loosely) keep track of deployment notes, in the main story (so it can be handled at release time)? This GRANT + CREATE SYNONYM would be a good thing to note so we don't forget it when deploying to a new environment.